src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -44,15 +46,14 @@ export const UserDetailsForm: React.FC<Props> = ({
   onSubmit,
 }) => {
   const textKeys = useTextKeys()
-  const [email, reallySetEmail] = React.useState(() => initialEmail)
-  const [emailError, setEmailError] = React.useState<boolean>(false)
-  const [ssn, reallySetSsn] = React.useState(() => initialSsn)
-  const [emailChangeTimout, setEmailChangeTimout] = React.useState<
-    number | null
-  >(null)
-  const [ssnChangeTimout, setSsnChangeTimout] = React.useState<number | null>(
+  const [email, reallySetEmail] = useState(() => initialEmail)
+  const [emailError, setEmailError] = useState<boolean>(false)
+  const [ssn, reallySetSsn] = useState(() => initialSsn)
+  const [showCreditCheckInfo, setShowCreditCheckInfo] = useState(false)
```

Comment:
Sorry, third comment (temporally, not on the page) about naming of booleans 🙃  See comment below for details

Reply:
And I know some other variables in this file don't follow this (looking at you old `emailError`!) but I guess that could be changed now as well, you know code scouting etc.